sdk/go/common/workspace/plugins_test.go

```diff
@@ -541,6 +541,32 @@ func TestPluginDownload(t *testing.T) {
 			assert.Equal(t, expectedBytes, readBytes)
 		})
 	})
+	t.Run("Test Downloading From GitLab Releases", func(t *testing.T) {
+		t.Setenv("GITLAB_TOKEN", token)
+		version := semver.MustParse("1.23.4")
+		spec := PluginSpec{
+			PluginDownloadURL: "gitlab://gitlab.com/278964",
+			Name:              "mock-gitlab",
+			Version:           &version,
+			Kind:              PluginKind("resource"),
+		}
+		source, err := spec.GetSource()
+		assert.NoError(t, err)
```

Comment:
should be require here, probably, since the rest of the test can't proceed if this fails.
```suggestion
		require.NoError(t, err)
```